include/fcl/config.h.in

```diff
@@ -45,4 +45,7 @@
 #cmakedefine01 FCL_HAVE_FLANN
 #cmakedefine01 FCL_HAVE_TINYXML
 
+#cmakedefine01 BUILD_TYPE_DEBUG
+#cmakedefine01 BUILD_TYPE_RELEASE
```

Comment:
Hm...maybe we should prefix with `FCL_`? ie. `FCL_BUILD_TYPE_DEBUG`, `FCL_BUILD_TYPE_RELEASE`?